tests/foreman/cli/test_oscap.py

```diff
@@ -10,31 +10,46 @@
 
 :CaseImportance: High
 
+:CaseAutomation: Automated
+
 :Upstream: No
 """
 from fauxfactory import gen_string
-
+from os import path
```

Comment:
please use (so that it will be always clean that path is from python os package)
```python
import os
```
instead of
```python
from os import path
```
also importing order separated by a line
 1. python packages
 2. external or third party packages
 3. project packages

eg:
```python
import os

from fauxfactory import gen_string

from robottelo import ssh
from robottelo.cli.base import CLIReturnCodeError
...



```